### ISSUE TITLE
Submission @thatkidmadrid

### ISSUE THREAD

**Issue by adrianna16** [adrianna16/kraken-good-accessibility#4]
I took away a div and created an hgroup instead, I also added an alt text to the image. I genuinely do not understand whatelse I need to add. im trying so hard to follow along in class I do not understand why I am not getting it. @thatkidmadrid

**Comment by thatkidmadrid** [comment]
That's a good catch, but there but still some articles that could  be used differently.

You're using a nav2 as a role; this is not a valid landmark role, you can just omit the role for any navigation that is not the primary navigation.

also, try using voiceover with the navigation links and you will see what I'm hinting at for the labels.

Don't get discouraged, just try your best, and the more specific your questions, the more specific my answer will be!

**Comment by adrianna16** [comment]
i think i get what you were saying?  i used the voice over and i adjusted my aria-labels and put role=presentation onto my ul elements so it doesnt state that is a list.
________________________________

**Comment by thatkidmadrid** [comment]
I'll take another look later see you in class!

**Comment by adrianna16** [comment]
Thank you, see you!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
